index.html

```diff
@@ -0,0 +1,368 @@
+
+![](images/postures.png)
+*<br>Example postures from left to right: “closed”, “laptop”, “table top”, “flat”, “canvas”, and “book”. ([Source](TODO))*
+
+### Spanning
+
+<dfn>Spanning</dfn> is a concept that allows apps or websites to run side by side on a [=dual-screen device=]. The ability to maximize a given application across the two [=screens=] is enabled through the [=window manager=]. Because spanning across the two screens double the screen estate it is expected that applications and websites leverage that additional space to optimize or enhance the UX to provide a better experience.
+
+### Window segments
+
+A <dfn>window segment</dfn> represents the regions (and their dimensions) of the window that reside on separate (adjacent) [=displays=].
+
+### Window interface
+
+The [=Window interface=] represents a window containing a DOM document, with its document property pointing to the DOM document. The Window interface hosts a number of globally available functions and namespaces, objects, and constructors associated with the concept of a user interface window. In tabbed browsing, each tab is represented by its own Window object.
+
+### Viewport
+
+<div class="issue">
+Ensure [Visual Viewport API](#visual-viewport-api) defines the two conceptual viewports, colloquially known as the [=visual viewport=] and [=layout viewport=], in a manner compatible with the [related web specifications](#related-web-specifications).
+</div>
+
+### Fold
+
+<dfn>Fold</dfn> is used as both a verb and a noun in the context of this explainer. _To fold_ means to lay one [part of the] screen over another [part of the] screen. _A fold_ (aka a screen fold) as a noun refers to the flexible [=screen area=] that bends.
+
+### CSS pixel
+
+A <dfn>CSS pixel</dfn> is denoted in CSS with the suffix px. It is a unit of length which roughly corresponds to the width or height of a single dot that can be comfortably seen by the human eye without strain ([MDN](https://developer.mozilla.org/en-US/docs/Glossary/CSS_pixel)).
+
+One CSS pixel doesn’t necessarily correspond to one physical screen pixel, since that would make everything very small on high resolution screens. Instead one such CSS pixel might correspond to for example 2x2 square of physical pixels - or even non-discrete values given a subpixel (red, green, and blue) layout.
+
+Many OS’s allow to change the scaling of all UI elements which often is reflected in the “device pixel ratio” that makes up one CSS pixel. A browser also often allows the user to apply a scale on top of the OS-provided scale on a per site basis. The layout is laid out considering the browser viewport in CSS pixels. For example, given a browser scale of 125%, a 800px wide viewport becomes a 640px wide viewport.
```

Comment:
link to devicepixelratio on mdn?

Reply:
Added https://drafts.csswg.org/cssom-view/#dom-window-devicepixelratio